setup.cfg

```diff
@@ -13,3 +13,9 @@ include_trailing_comma = true
 known_first_party = cms, menus
 multi_line_output = 5
 not_skip = __init__.py
+
+
+[options]
+install_requires =
```

Comment:
I don't know if this will conflict with the `install_requires` in `setup.py`

Reply:
Yeah, it seems the thing people do these days is to have a very simple `setup.py` and all their options defined in `setup.cfg`. It's just quite a big job to move everything over.